### PR TITLE
Extended support for API 2400

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Extended support to API 2400 for following resources

- Ethernet Network
- Enclosure
- Enclosure Group
- FC Network
- FCOE Network
- Hypervisor Cluster Profiles
- Hypervisor Manager
- Interconnects
- Interconnect Types
- Logical Enclosure
- Network Set
- Scope
- Server Certificates
- Server Hardware
- Server Hardware Types
- Server Profile
- Server Profile Template
- Storage Pool
- Storage System
- Storage Template
- Storage Volume
- Storage Volume Attachments
- Tasks
- Uplink Set 

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
-# [v1.7.1]
+# [v1.8.0] (Unreleased)
+### Notes
+- This release extends supports of the SDK to Oneview API Version 2400.
+- Minor Bug Fixes
```
Comment (VenkateshRavula) [inline CHANGELOG.md:7]: Since you already mentioned bugs & enhancements below, is it required to mention it here also..?

Comment (AsisBagga) [reply CHANGELOG.md:7]: following similar pattern in previous releases.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,37 @@
...
+- Ethernet Network
+- Enclosure
+- Enclosure Group
+- FC Network
+- FCOE Network
+- Hypervisor Cluster Profiles
+- Hypervisor Manager
+- Interconnects
+- Interconnect Types
+- Logical Enclosure
```
Comment (VenkateshRavula) [inline CHANGELOG.md:19]: LI & LIG are missing here. Are they part of other PR..?

Comment (AsisBagga) [reply CHANGELOG.md:19]: yes

**Review by VenkateshRavula — APPROVED** [review]

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**